index.php

```diff
@@ -0,0 +1,30 @@
+<?php include('connect.php'); ?><!DOCTYPE html>
```

Comment:
Not a big deal, but I would put these on their own line.  All PHP blocks will be deleted before shipping to server, so the Doctype will still be at the top.  Even if it's not, the browser ignores whitespace.